doc/whats_new.rst

```diff
@@ -1279,6 +1279,9 @@ Model evaluation and meta-estimators
      the parameter ``n_labels`` is renamed to ``n_groups``.
      :issue:`6660` by `Raghav RV`_.
 
+   - The :mod:`sklearn.linear_model.randomized_l1` is deprecated.
+     :issue: `8995` by `Andreas Mueller`_.
```

Comment:
The "by" refers to the contributor, so you. You can use the ``:user:`` pattern as elsewhere in whatsnew.